source/user-manual/wazuh-dashboard/management.rst

```diff
@@ -0,0 +1,99 @@
+.. Copyright (C) 2015, Wazuh, Inc.
+
+.. meta::
+  :description: The Wazuh dashboard gives you a quick view of your agents, alerts, and cluster. Learn how to configure its features in this section. 
+  
+.. _dashboard_management:
+
+Dashboard management
+========================
+
+The **Dashboard management** section allows you to configure and customize your Wazuh dashboard experience, as well as seeing the generated reports and the application logs.
```

Comment:
Maybe a screenshot of the Dashboard management options could give a graphic overview of all the available options

![imagen](https://github.com/wazuh/wazuh-documentation/assets/47069802/fcac2aa7-c511-4c42-8bd3-ed1e5f69c427)

or maybe a local contents table for this.

```
.. contents::
   :local:
   :depth: 1
   :backlinks: none
```

Reply:
Solved in 32d95d828e5667d6c5bf02462bd1e42080c86723